docs/pages/modules/use-standalone-expo-module-in-your-project.mdx

```diff
@@ -65,37 +79,43 @@ Compile and run the app with the following command:
   ]}
 />
 
-You can now use the module inside your app. To test this, let's edit the **App.js** file in the app and render the text message from the `expo-settings` module:
+You can now use the module in your app. To test it, edit the **App.js** file in your app and render the text message from the `expo-settings` module:
 
 ```jsx App.js
-import * as Settings from 'expo-settings';
-import { StatusBar } from 'expo-status-bar';
 import React from 'react';
 import { Text, View } from 'react-native';
+import * as Settings from 'expo-settings';
 
 export default function App() {
   return (
     <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
       <Text>{Settings.hello()}</Text>
-      <StatusBar style="auto" />
     </View>
   );
 }
 ```
 
-After this configuration, you'll see a text that says "Hello world! 👋" in the app.
+After this configuration, the app displays the text "Hello world! 👋".
+
+</Step>
 
 ## Publish the module to npm
 
-You can publish the module on npm and then add it to your project by installing it as a dependency.
+Publish the module on npm, then install it as a dependency in your project.
```

Comment:
```suggestion
You can publish the module on npm and install it as a dependency in your project by following the steps below.
```

Not an instruction but an explanation of what people can do by following the steps below.